specification/validated-relationships-service-api.yaml

```diff
@@ -657,24 +562,24 @@ paths:
         Common update scenarios include:
         * Revocation of the role (status change from "active" to "inactive")
         * Changing the legal basis of the role (TBC)
-        * Updating the end date for timebound access
+        * Updating the end date for time-bound access
 
         ## Request Requirements
         * The proxy role must exist and be identified by a valid identifier
         * JSON Patch operations must be valid according to RFC 6902
         * Status changes must use valid status codes from <http://hl7.org/fhir/consent-state-codes>
 
         ## Sandbox test scenarios
-        | Scenario | Request | Response |
-        | -------- | ------- | -------- |
-        | Successful status update | Valid patch changing status to 'active' | HTTP Status 200 and OperationOutcome|
-        | Successful access level update | Valid patch modifying the role end date | HTTP Status 200 and OperationOutcome |
-        | Multiple valid changes | Valid patch with multiple operations | HTTP Status 200 and OperationOutcome |
-        | Invalid patch format | Malformed JSON patch document | HTTP Status 400 and INVALID_PATCH_FORMAT error response |
-        | Invalid path | Patch targeting non-existent element | HTTP Status 400 and INVALID_PATCH_PATH error response |
-        | Invalid status code | Patch with invalid status value | HTTP Status 422 and INVALID_STATUS_CODE error response |
-        | Resource not found | Patch for non-existent Consent | HTTP Status 404 and RESOURCE_NOT_FOUND error response |
-        | Invalid state transition | Patch attempting invalid status change | HTTP Status 422 and INVALID_STATE_TRANSITION error response |
+        | Scenario                        | Request                                   | Response                                                      |
+        | ------------------------------- | ----------------------------------------- | ------------------------------------------------------------- |
```

Comment:
Can key values be added to the request so it's obvious how someone can trigger each request